### ISSUE TITLE
Consider applying to GNOME Circle

### ISSUE THREAD

**Issue by badcel** [gircore/gir.core#231]
If we have some releases we can consider applying to [GNOME Circle](https://circle.gnome.org/).

**Comment by tauin** [comment]
this week in gnome (which generally gets a lot of views, from being posted on various websites) has a "Circle Apps and Libraries" section, maybe it would be worth applying so you could attract potential contributors?

**Comment by badcel** [comment]
Good idea. But I think we at least need some beta release / nuget package before posting some news: https://gitlab.gnome.org/World/twig/-/blob/main/README.md

**Comment by tauin** [comment]
true; additionally, looking over [Gnome Circles requirements](https://gitlab.gnome.org/Teams/Circle/-/blob/master/README.md#who-can-apply) there are requirements for libraries being admitted like documentation and actually being used by apps (although the latter seems a big vague, I doubt a simple example program like the ones located in the samples directory would count)

**Comment by badcel** [comment]
Yeah we will see. I hope that we can get into a circle app even if there are no big apps using the code base to gain some visibility.
